services/blockchain-connector/config.js

```diff
@@ -112,4 +112,11 @@ config.job = {
 	},
 };
 
+config.verifyClientConnection = {
+	// Every n number of blocks, verify if client connection is alive
+	numOfBlocksVerify: process.env.NUM_BLOCKS_VERIFY || 15,
```

Comment:
Just a preferential suggestion, would something like `connectionVerifyBlockInterval` be more suitable naming?